lbrynet/daemon/Downloader.py

```diff
@@ -99,12 +97,15 @@ def set_status(self, status, name):
 
     @defer.inlineCallbacks
     def check_fee_and_convert(self, fee):
-        max_key_fee_amount = self.convert_max_fee()
+        currency, amount = self.max_key_fee['currency'], self.max_key_fee['amount']
+        max_key_fee_amount = self.convert_max_fee(currency, amount)
         converted_fee_amount = self.exchange_rate_manager.convert_currency(fee.currency, "LBC",
                                                                            fee.amount)
+        is_disabled_max_key_fee = True if amount < 0 else False
+
         if converted_fee_amount > (yield self.wallet.default_account.get_balance()):
             raise InsufficientFundsError('Unable to pay the key fee of %s' % converted_fee_amount)
-        if converted_fee_amount > max_key_fee_amount and not self.disable_max_key_fee:
+        if converted_fee_amount > max_key_fee_amount and not is_disabled_max_key_fee:
```

Comment:
This is really verbose and redundant:
```
is_disabled_max_key_fee = True if amount < 0 else False
```
above can be represented much more succinctly as:
```
is_disabled_max_key_fee = amount < 0
```
but since the `is_disabled_max_key_fee` isn't used anywhere else and is making a double negative `(not amount < 0)` you could make this much cleaner and easier to read by removing the `not` and flipping the less than to greater than:
```
 if converted_fee_amount > max_key_fee_amount and amount > 0:
```
unless I'm missing something in the logic...

Reply:
No, you are not missing anything. It makes complete sense.